src/components/User/UserFeedbackComponent.jsx

```diff
@@ -15,6 +15,7 @@ import ActionComponent from '../ActionComponent';
 import TableHeaderComponent from '../common/TableHeaderComponent';
 
 import '../../_css/UserFeedback.css';
+import { ItemsNotFoundComponent } from '../ItemsNotFoundComponent';
```

Comment:
This should be a default import. The same comment applies to all areas where `ItemsNotFoundComponent` is imported